rand_distr/Cargo.toml

```diff
@@ -20,9 +20,17 @@ travis-ci = { repository = "rust-random/rand" }
 appveyor = { repository = "rust-random/rand" }
 
 [dependencies]
-rand = { path = "..", version = "0.7" }
+rand = { path = "..", version = "0.7", default-features = false }
+num-traits = { version = "0.2", default-features = false, features = ["libm"] }
+
+[features]
+default = ["std"]
+std = ["rand/std", "num-traits/std", "alloc"]
+alloc = ["rand/alloc"]
```

Comment:
Strictly speaking we are not using functionality dependent on `alloc` and `std` features in those dependencies, so I am not sure if we should forward these features. I think it's a common convention, but should we consciously follow it?

Reply:
Potentially this could add an unwanted `getrandom` dependency, so there is *some* argument to reducing dependencies (though normally they would be used anyway).

Reply:
I've removed the forwarding of those features.